spec/lib/myfinance/resources/person_spec.rb

```diff
@@ -0,0 +1,226 @@
+        expect(subject.collection.first.address).to eq("")
+        expect(subject.collection.first.address_number).to eq("")
+        expect(subject.collection.first.city).to eq("")
+        expect(subject.collection.first.complement).to eq("")
+        expect(subject.collection.first.country).to eq("")
+        expect(subject.collection.first.created_at).to eq(DateTime.parse("2015-11-12T14:34:37-02:00"))
+        expect(subject.collection.first.customer).to eq(true)
+        expect(subject.collection.first.email).to eq("")
+        expect(subject.collection.first.excel_import_id).to be_nil
+        expect(subject.collection.first.febraban_favored_code).to be_nil
+        expect(subject.collection.first.febraban_name).to be_nil
+        expect(subject.collection.first.federation_subscription_number).to eq("42.308.611/0001-41")
+        expect(subject.collection.first.federation_subscription_number_only_numbers).to eq("42308611000141")
+        expect(subject.collection.first.federation_subscription_type_id).to be_nil
+        expect(subject.collection.first.force_destroy).to eq(false)
+        expect(subject.collection.first.guid).to be_nil
+        expect(subject.collection.first.imported_from_sync).to eq(false)
+        expect(subject.collection.first.interested_users_ids).to eq([3691])
+        expect(subject.collection.first.modified_by_sync).to eq(false)
+        expect(subject.collection.first.name).to eq("Myfreecomm")
+        expect(subject.collection.first.neighborhood).to eq("")
+        expect(subject.collection.first.note).to eq("")
+        expect(subject.collection.first.person_type).to eq("JuridicalPerson")
+        expect(subject.collection.first.phone).to eq("")
+        expect(subject.collection.first.site).to eq("")
+        expect(subject.collection.first.state).to eq("")
+        expect(subject.collection.first.supplier).to eq(true)
+        expect(subject.collection.first.updated_at).to eq(DateTime.parse("2015-11-12T14:34:37-02:00"))
+        expect(subject.collection.first.use_count).to eq(0)
+        expect(subject.collection.first.zip_code).to eq("")
+      end
+    end
+
+    context "when not found" do
+      let(:client) { Myfinance.client("") }
+      subject { client.people.find_all }
+
+      it "raises NotFound" do
+        expect { subject }.to raise_error(Myfinance::RequestError)
+      end
+    end
+  end
+
+  describe "#find", vcr: true do
+    context "when success" do
+      subject { client.people.find(199424) }
+
+      it "returns a find_all of people" do
+        expect(subject.class).to eq(Myfinance::Entities::Person)
+        expect(subject.id).to eq(199424)
+        expect(subject.address).to eq("")
+        expect(subject.address_number).to eq("")
+        expect(subject.city).to eq("")
+        expect(subject.complement).to eq("")
+        expect(subject.country).to eq("")
+        expect(subject.created_at).to eq(DateTime.parse("2015-11-12T14:34:37-02:00"))
```

Comment:
Do not use `DateTime.parse` without zone. Use one of `Time.zone.parse`, `DateTime.parse.current`, `DateTime.parse.in_time_zone`, `DateTime.parse.utc`, `DateTime.parse.getlocal`, `DateTime.parse.iso8601`, `DateTime.parse.jisx0301`, `DateTime.parse.rfc3339`, `DateTime.parse.to_i`, `DateTime.parse.to_f` instead.